### ISSUE TITLE
Make it possible to stop trying to connect when pressing Play Online/Local

### ISSUE THREAD

**Issue by AmonShokhinAhmed** [lizrad/recursio#185]
If the connection fails this stalls the player for a very long time, which is pretty annoying if the button was pressed on accident.
